src/index.js

```diff
@@ -168,28 +97,9 @@ Image.defaultProps = {
 const cards = variant({ key: 'cards' })
 
 export const Card = styled(Box)(
-  borders,
-  borderColor,
-  borderRadius,
-  boxShadow,
-  backgroundImage,
-  backgroundSize,
-  backgroundPosition,
-  backgroundRepeat,
-  opacity,
+  border,
```

Comment:
Should this be using `compose` here as suggested by the migration guide?

https://styled-system.com/guides/migrating#compose